plugin-info-vue-full-app/plugin/manifest.json

```diff
@@ -0,0 +1,64 @@
+{
+  "id": "com.adobe.uxp.info.plugin",
+  "name": "Plugin Info",
+  "version": "1.0.0",
+  "main": "index.html",
+  "manifestVersion": 4,
+  "host": {
+    "app": "PS",
```

Comment:
This plugin would definitely work well as a cross-compatible plugin with Adobe XD. Is there a way to indicate both `PS` and `XD` in the manifest?